content/3_Silver/Func_Graphs.problems.json

```diff
@@ -55,6 +55,18 @@
         "kind": "internal"
       }
     },
+    {
+      "uniqueId": "cses-3398",
+      "name": "Permutation Rounds",
+      "url": "https://cses.fi/problemset/task/3398",
```

Comment:
hm doesn't this involve topics from https://usaco.guide/gold/divisibility that haven't been introduced yet

Reply:
but sieve has literally appeared in silver...

Reply:
that problem isn't actually in any of the USACO Guide Silver modules though

Reply:
moved to the more relevant module